### PR TITLE
SONARIAC-569 Rule S6502: Disabling builder sandboxes is security-sensitive

### PR DESCRIPTION
**Pull request by nils-werner-sonarsource**

### CODE REVIEW

**Review by rudy-regazzoni-sonarsource — APPROVED** [review]
LGTM ! Please see my comment.

**Comment by nils-werner-sonarsource** [comment]
> LGTM ! Please see my comment.

@rudy-regazzoni-sonarsource there is no comment

File: iac-extensions/docker/src/test/resources/checks/BuilderSandboxCheck/BuilderSandboxCheck.dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Noncompliant@+1 {{Make sure that disabling the builder sandbox is safe here.}}
+RUN --security=insecure cat /proc/self/status > /output.txt
+#   ^^^^^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+RUN --security=insecure ["cat", "/proc/self/status > /output.txt"]
+
+RUN --security=sandbox cat /proc/self/status > /output.txt
+RUN cat /proc/self/status > /output.txt
+RUN --network=none pip install --find-links wheels mypackage
```
Comment (rudy-regazzoni-sonarsource) [inline iac-extensions/docker/src/test/resources/checks/BuilderSandboxCheck/BuilderSandboxCheck.dockerfile:13]: Possible other usecase :
```
--SECURITY=insecure
--security=INSECURE
ARG security=insecure
--security=$security
```
Not necessary, I let you decide.

Comment (nils-werner-sonarsource) [reply iac-extensions/docker/src/test/resources/checks/BuilderSandboxCheck/BuilderSandboxCheck.dockerfile:13]: Good point

**Pull request merged**